### ISSUE TITLE
feature: Allow "token label" to be the "Triggered by" value in Runs dashboard

### ISSUE THREAD

**Issue by ChrisEdgington** [windmill-labs/windmill#4169]
We are using different tokens generated by the same user account for each warehouse computer that will be accessing our windmill routes. However, this makes it impossible to see what windmill activity is specific to a computer. So if the operator of PC4 says, "Hey, I had a problem processing a shipment this morning", to debug via the windmill Runs dashboard, we need to sort through all the executions. It would be helpful to be able to filter or sort based on the token label that was provided when the token was created. With this feature, if we create 4 tokens, with labels "WH01, WH02, ConfRoomUAT, ConfRoomCFG", we could easily see only the runs for the conference room user-acceptance-testing by filtering on the ConfRoomUAT token label.

<img width="1858" alt="image" src="https://github.com/user-attachments/assets/8de4d081-8dbd-4562-9720-5f00ef51393f">

**Comment by HugoCasa** [comment]
What version are you on? We now display the token label on this page and you can filter runs by this value.

**Comment by ChrisEdgington** [comment]
EE v1.355.4-2-g05fa60e68
Our self-hosted deployment is managed by IT group - what version should I ask them to upgrade us to?

**Comment by HugoCasa** [comment]
@ChrisEdgington it's available since v1.356 but i suggest upgrading to latest

**Comment by ChrisEdgington** [comment]
@HugoCasa We are now running EE v1.377.1-13-gc288d73c1 and the only reference to "label" I see on the Runs dashboard is in the filter menu at the top, and the info box tooltip does not seem to indicate that this label has anything to do with the token. Definitely I do not see that the token label is available as an optional display parameter for the "triggered by" column, which is what this feature request was for.

**Comment by ChrisEdgington** [comment]
@HugoCasa I did confirm that now when a run is driven by a token that includes a label, that label shows up in the "Triggered by" column. I also confirmed that if I click on that label, my runs get filtered to include only those with that token label. However, I could not get run filtering on token label to work by clicking Label in the filter buttons then typing a label. Is this expected? I tried entering a label with and without the "label-" prefix that shows up in the column.
